### PR TITLE
Styling alton payment page

### PR DESCRIPTION
**Pull request by davidallardsb**
Jira: https://hxshortbreaks.atlassian.net/browse/FP-2267

What does this PR do? (please provide any background)
Maps the relevant alton brand styles to the payment page.

What tests does this PR have?
None

How can this be tested?

1. Npm link FP-2267 brands repo from the-wall FP-2267
2. Head to the alton payment page using this bit: https://www.altontowersholidays.com/?bits%5Bsb_altonseosbp%5D=1
3. Make sure the page works & looks OK on all devices 

Any tech debt?
Nope 

Screenshots / Screencast

booking-alton-local holidayextraspartners co uk-payment-tag upgrades basketid 03af5ba487ad21f6dd4943879e3f56e0 versionid c8864552 agent ato01 customercode q roomrates 5bhotelcode 5d resspl roomrates 5bcheckindate 5d 2018-06-13 roomrates 5bb

#### What gif best describes how you feel about this work?

- [ ] I have checked our general [contributing document](https://github.com/holidayextras/culture/blob/master/CONTRIBUTING.md) and the project specific [contributing document](../blob/master/CONTRIBUTING.md) (if present) and I'm happy for this to be reviewed.

---

**Reviewer**
- [ ] :+1:
- [ ] UXUI Team :+1:
- [ ] I don't think this PR needs any additional reviewers

By adding a +1 you are confirming you have...
- Witnessed the work behaving as expected (this could be on the author's machine or screencast).
- Ensured the release has the correct version bump.
- Checked all the work locally on pattern library.

### CODE REVIEW

File: brands/alton-towers/src/less/colours.less [hunk 1]
```diff
@@ -7,10 +7,10 @@
 // Canvas colors
 @color-white:              rgb(255,255,255);
 @color-black:              rgb(0,0,0);
-@brand-white-off:          rgb(243, 244, 248); //#f3f4f8
+@brand-white-off:          rgb(243, 244, 248);   //#f3f4f8
 @color-gold:               rgb(210, 186, 87);    //#d2ba57 Yellow
-@color-red:                rgb(226, 0, 26);    //#e2001a Red
-@color-blue:               rgb(33, 28, 91);    //#211c5b Blue
+@color-red:                rgb(226, 0, 26);      //#e2001a Red
+@color-blue:               rgb(33, 28, 91);      //#211c5b Blue
```
Comment (putvande) [inline brands/alton-towers/src/less/colours.less]: 👍

File: brands/alton-towers/src/less/navbar.less [hunk 2]
```diff
@@ -101,11 +107,12 @@
   }
   .navbar-brand {
     width: @logo-width-lg;
-    height: (@logo-height-lg + 15);
+    height: (@logo-height-lg);
     background-size: @logo-width-lg @logo-height-lg; // Specificity for Bootstrap reset
     &,
     .navbar-static-top>.container-fluid & {
```
Comment (putvande) [inline brands/alton-towers/src/less/navbar.less]: Can you put some spacing around these guys. `.navbar-static-top > .container-fluid`

File: brands/alton-towers/src/less/navbar.less [hunk 3]
```diff
@@ -101,11 +107,12 @@
...
-      margin-left: 0;
+      // Set's the logo in the perfect place :)
```
Comment (putvande) [inline brands/alton-towers/src/less/navbar.less]: We should put more smiley faces in our code :)

File: brands/alton-towers/src/less/progress-trackers.less [hunk 4]
```diff
@@ -1,14 +1,14 @@
...
-  font-weight: normal;
-  >li {
-    color: @text-color;
-    >a:not(.btn):not(.list-group-item):not(.panel-title):not(.alert-link):not([role=tab]) {
-      &,
-      &:hover{
-        color: @brand-primary;
-      }
+
+.panel > nav > ol.progress-tracker {
```
Comment (putvande) [inline brands/alton-towers/src/less/progress-trackers.less]: Why has this been made more specific? Does Bootstrap override this?

File: brands/alton-towers/src/less/variables.less [hunk 5]
```diff
@@ -108,21 +108,21 @@
...
 //##
-@logo:                             'https://www.altontowers.com/img/logos/alton_towers_logo_large.png';
+
+@logo:                             '@{brand-asset-path-images}alton_towers_logo_medium.png';
 @logo-org-width:                   310px;
 @logo-org-height:                  259px;
-@logo-width:                       ceil(@logo-org-width/4); // ~25%
-@logo-height:                      ceil(@logo-org-height/4); // ~25%
+@logo-width:                       108px; // ~25%
+@logo-height:                      90px; // ~25%
```
Comment (putvande) [inline brands/alton-towers/src/less/variables.less]: Why don't these have calculations like before? Also, the comment behind it doesn't match anymore ;)

File: brands/alton-towers/src/less/progress-trackers.less (hunk 4 above)
Comment (davidallardsb) [reply brands/alton-towers/src/less/progress-trackers.less]: progress-tracker-justified was overriding this but I added a better solution :)

File: brands/alton-towers/src/less/variables.less (hunk 5 above)
Comment (davidallardsb) [reply brands/alton-towers/src/less/variables.less]: The calculations did not generate the logo in the correct place so I thought it was best to hard code the correct size. I have removed the comment :)

**Pull request merged**